read-fonts/src/font_data.rs

```diff
@@ -179,6 +179,34 @@ impl<'a> Cursor<'a> {
         self.pos += n_bytes;
     }
 
+    /// Read a variable length u32 and advance the cursor
+    pub(crate) fn read_u32_var(&mut self) -> Result<u32, ReadError> {
```

Comment:
Is this particular variable-length u32 encoding going to be used in more than just Varc? If not, I think this logic would make more sense as a function in the varc module... I think it's higher-level than the cursor itself, and the logic is easy to implement without really needing any access to cursor internals (anything with access to  `&mut cursor` can handle it.)


not a big deal in any case, but it feels _slightly_ out of place.

Reply:
Interesting. To me it's just like any of the cursor ops and fits more naturally there than anywhere else but I guess it's somewhat subjective. Fwiw I had it in varc and had _exactly_ the same feeling: it felt slightly out of place.